addons/l10n_pk/demo/account_demo.py

```diff
@@ -0,0 +1,324 @@
+                    'invoice_user_id': 'base.user_demo',
+                    'invoice_payment_term_id': 'account.account_payment_term_end_following_month',
+                    'invoice_date': fields.Datetime.today() - relativedelta(days=2),
+                    'invoice_line_ids': [
+                        Command.create({
+                            'product_id': 'product.product_product_8',
+                            'quantity': 4,
+                            'price_unit': 1000.0,
+                            'tax_ids': [Command.set([
+                                _get_tax_by_id('pk_sales_tax_17')
+                            ])],
+                        }),
+                        Command.create({
+                            'product_id': 'product.product_product_7',
+                            'quantity': 3,
+                            'price_unit': 2000.0,
+                            'tax_ids': [Command.set([
+                                _get_tax_by_id('pk_sales_tax_services_10')
+                            ])],
+                        }),
+                    ]
+                },
+                'demo_vendor_bill_3': {
+                    'ref': 'INV/003',
+                    'move_type': 'in_invoice',
+                    'partner_id': 'res_partner_islamabad',
+                    'invoice_user_id': 'base.user_demo',
+                    'invoice_payment_term_id': 'account.account_payment_term_end_following_month',
+                    'invoice_date': fields.Datetime.today() - relativedelta(days=3),
+                    'invoice_line_ids': [
+                        Command.create({
+                            'product_id': 'product.product_product_9',
+                            'quantity': 1,
+                            'price_unit': 1000.0,
+                            'tax_ids': [Command.set([
+                                _get_tax_by_id('pk_sales_tax_services_17')
+                            ])],
+                        }),
+                        Command.create({
+                            'product_id': 'product.product_product_10',
+                            'quantity': 1,
+                            'price_unit': 2000.0,
+                            'tax_ids': [Command.set([
+                                _get_tax_by_id('pk_sales_tax_services_10'),
+                                _get_tax_by_id('pk_sales_tax_services_1')
+                            ])],
+                        }),
+                    ]
+                },
+            }
+        else:
+            return super()._get_demo_data_move(company)
+
+    @api.model
+    def _get_demo_data_config_settings(self, company=False):
+        return {
+            'sales_credit_limit': {
+                'account_use_credit_limit': True,
+                'account_default_credit_limit': '10000'
+            }
+        }
+
+    def _post_load_demo_data(self, company=False):
+        if company.account_fiscal_country_id.code == "PK":
```

Comment:
```suggestion
        if company.account_fiscal_country_id.code != "PK":
            return super()._post_load_demo_data(company)
```
To make code readability better you might have to do a less indentation if you return the super at the initiation of the method apply the same in other methods